recipe/meta.yaml

```diff
@@ -1,69 +1,51 @@
-{% set version = "0.12.0" %}
-{% set cov_fail_under = "82" %}
-{% set min_sphinx = "4.2" %}
-
+{% set name = "pydata-sphinx-theme" %}
+{% set version = "0.16.1" %}
 package:
   name: pydata-sphinx-theme
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/py3/p/pydata-sphinx-theme/pydata_sphinx_theme-{{ version }}-py3-none-any.whl
-    sha256: c17dbab67a3774f06f34f6378e896fcd0668cc8b5da1c1ba017e65cf1df0af58
-  - folder: src
-    url: https://github.com/pydata/pydata-sphinx-theme/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 1759e12524b440d136228086206fbd23c3cf8007a64d3cecd7eda492f85617d3
-
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydata_sphinx_theme-{{ version }}.tar.gz
+  sha256: a08b7f0b7f70387219dc659bff0893a7554d5eb39b59d3b8ef37b8401b7642d7
 build:
   number: 0
-  noarch: python
+  skip: true   #[py<39]
   script:
-    - {{ PYTHON }} -m pip install --find-links . --no-index --ignore-installed --no-deps pydata-sphinx-theme
-    - cd {{ RECIPE_DIR }} && {{ PYTHON }} collect_licenses.py
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - sphinx-theme-builder 0.2.0b2
   run:
+    - sphinx >=6.1
     - beautifulsoup4
     - docutils !=0.17.0
-    - packaging
+    - babel
     - pygments >=2.7
-    - python >=3.7
-    - sphinx >={{ min_sphinx }}
+    - accessible-pygments
+    - typing-extensions
+    - packaging
+    - python
 
 test:
-  source_files:
-    - src/tests
   imports:
     - pydata_sphinx_theme
   commands:
     - pip check
-    - cd src/tests
-    - pytest -vv --cov=pydata_sphinx_theme --cov-branch --cov-report=term-missing:skip-covered --no-cov-on-fail --cov-fail-under={{ cov_fail_under }}
```

Comment:
What was the issue with tests? Missing `pydata-sphinx-theme` test dependency?

Reply:
It was agreed that some of these tests were not needed nor were some of these dependencies as the source URL when routed to pypi's version did not include many of the tests that github's upstream version included. This package is also not ready for review as of yet as a patch will be implemented for the build failures for linux! Thank you for taking a look at though!